rdr_service/offline/main.py

```diff
@@ -1308,6 +1314,13 @@ def _build_pipeline_app():
         methods=['GET']
     )
 
+    offline_app.add_url_rule(
+        OFFLINE_PREFIX + 'GenomicNotifyAppointmentGRoRRevoked',
```

Comment:
Can you change this to GenomicNotifyAppointmentGRoRChanged to match the names of the functions that are called?

Reply:
Yes, this has been fixed.